### DISCUSSION TITLE
Power switch for Tesla Android

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#299] [Show and tell]
Tired of opening the center armrest for connecting/disconnecting Tesla Android, I designed a power switch that fits under the (pre-refresh) armrest. Switch is easily accessible with armrest closed.

[Tesla Android Switch APEM TL36P005000.zip](https://github.com/tesla-android/issue-tracker/files/12617838/Tesla.Android.Switch.APEM.TL36P005000.zip)

![20230915_082611](https://github.com/tesla-android/issue-tracker/assets/9882700/a6b2ff97-3ae1-4177-8a6c-d027cf5c9c98)
![20230915_082601](https://github.com/tesla-android/issue-tracker/assets/9882700/3795e3f3-fb0b-4820-be19-a66a17d9b5af)

**Comment by 8bitjer** [comment]
Can you link the switch? I have been looking for one that has enough power pass through for the pi.

**Comment by RetoHB** [comment]
The part number is in the file name. I am using the switch above its current specification. Let's see how long it will live...
